main/getting-started/ertp-introduction.md

```diff
@@ -12,83 +12,93 @@ object. If it doesn't have a reference, it can't.
 
 ## Creating assets with ERTP
 
-In ERTP, *mints* create digital *assets*. Access to an asset type's
-mint lets you create more digital assets of that type. You can then
+In ERTP, *mints* create *digital assets*. Access to an asset kind's
+`mint` lets you create more digital assets of that kind. You can then
 store new assets in a *payment* or a *purse*. 
-- **Payments**: Assets you intend to send to someone else.
-- **Purses**: Store assets until you withdraw them into a payment for use
+- `Payments`: Assets you intend to send to someone else.
```

Comment:
How about "to move between `purses` or to other destinations."  Because invitations are payments, and you don't put them in purses. And you escrow payments, which I don't think involves putting them in purses? I think the key thing amounts to "intend to transfer control of to some other party", but not sure if it'll be read correctly.

Reply:
Not always to some other party, but "to move between purses or to other destinations."  is fine.